packages/typings/command/index.d.ts

```diff
@@ -27,6 +27,7 @@ export interface Report {
   data?: ReportItem[];
   /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
   raw?: any;
+  rawReports?: Report[];
```

Comment:
Let's hold onto the individual reports. This should allow for the outputs to handle it for example letting the json one write all the reports to disk.